SimulationAgent/FileWrapper.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved. 
+
```

Comment:
psl move this file to Services, it might come handy in other scenarios

Reply:
done